perllib/FixMyStreet/Cobrand/Merton.pm

```diff
@@ -147,4 +147,15 @@ sub open311_post_send {
         delete $self->{original_external_id};
     }
 }
+
+sub open311_extra_data_include {
+    my ($self, $row, $h) = @_;
+
+    if (my $echo_id = $row->get_extra_field_value('echo_id')) {
```

Comment:
Not sure what this is for - all extra fields are included when sending via Open311 anyway, aren't they?